src/features/document/document_feature.py

```diff
@@ -78,6 +78,8 @@ def add_document(
       - Reference to package by id: PROTOCOL://DATA SOURCE/$ID
       - Reference to package by path: PROTOCOL://DATA SOURCE/ROOT PACKAGE/SUB PACKAGE
       The PROTOCOL is optional, and the default is dmss.
+
+    This endpoint can be used to add elements to a list attribute in an entity.
```

Comment:
In the reference tests you're using it to fill an empty object. Not sure what is correct use of the add endpoint

Reply:
I believe it should be both adding elements to list and filling empty object. Can update the docs